pkg/bbgo/interact.go

```diff
@@ -13,35 +12,9 @@ import (
 	"github.com/c9s/bbgo/pkg/types"
 )
 
-type PositionCloser interface {
-	ClosePosition(ctx context.Context, percentage fixedpoint.Value) error
-}
-
-type PositionReader interface {
-	CurrentPosition() *types.Position
-}
-
-type StrategyStatusProvider interface {
-	GetStatus() types.StrategyStatus
-}
-
-type Suspender interface {
-	Suspend(ctx context.Context) error
-	Resume(ctx context.Context) error
-}
-
-type StrategyController interface {
-	StrategyStatusProvider
-	Suspender
-}
-
-type EmergencyStopper interface {
-	EmergencyStop(ctx context.Context) error
-}
```

Comment:
these interfaces are good, I think we don't need to remove them?

Reply:
I thought callback style is better?

Reply:
nop~ I mean PositionCloser is for closing the position, it's implemented, we don't need a callback for it.

If we have a position object and know how much we want to close, we can close it directly without implementing the method. It's better to share the position closing code between different strategies, since they do the same thing.